log/keyvalue.go

```diff
@@ -321,3 +356,11 @@ func Map(key string, value ...KeyValue) KeyValue {
 func Empty(key string) KeyValue {
 	return KeyValue{key, Value{}}
 }
+
+// String returns key-value pair as a string, formatted like "key=value".
+//
+// The returned string is meant for debugging;
+// the string representation is not stable.
+func (a KeyValue) String() string {
+	return fmt.Sprintf("%s=%s", a.Key, a.Value)
```

Comment:
What about using a colon instead of an equal sign to not imply equality?

```suggestion
	return fmt.Sprintf("%s:%s", a.Key, a.Value)
```

Reply:
I agree. Especially that https://go.dev/play/p/PsJA9v9A2vy

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-go/pull/5117/commits/a55b71c78b48152828813fee230e957fd658fbea